scripts/generate-preact.js

```diff
@@ -17,9 +17,13 @@ export default ${component.name};
 
 declare const ${component.name}: MdiReactIconComponentType;
 export default ${component.name};
-`, () => `import { FunctionalComponent } from 'preact';
+`, () => `import { FunctionalComponent, HTMLProps } from 'preact';
 
-export interface MdiReactIconProps {
+type SVGProps = HTMLProps<SVGElement>
+
+type ReservedProps = 'size' | 'width' | 'height' | 'fill' | 'viewBox'
+
+export interface MdiReactIconProps extends Pick<SVGProps, Exclude<keyof SVGProps, ReservedProps>> {
```

Comment:
I just did some research and found out that basically there are no typings for SVG elements in the current stable release of Preact. This will improve with Preact 10.x which is still in beta. Please just revert the changes for the Preact generator since they break the current typings.

Reply:
Wow good catch. Reverted.